README.md

```diff
@@ -243,6 +283,10 @@ Known issues:
 
 ### How to upgrade from v6.x.x to v7.x.x?
 
+- Themes have been introduced and should be set as needed to retain the correct look and feel, see [theme documentation](#themes).
```

Comment:
(Rewrite to remove the passive voice)
- We have introduced themes which you should set as needed to retain the correct look and theme for your product, see [theme documentation](#themes).